### ISSUE TITLE
JITaaS compilation failures in JITaaS PR testing

### ISSUE THREAD

**Issue by harryyu1994** [eclipse/openj9#5746]
```
16:06:26  /home/jenkins/workspace/Build_JDK8_x86-64_linux_Personal/build/linux-x86_64-normal-server-release/vm/compiler/../compiler/rpc/J9Server.cpp:111:8: error: ‘SSL_CTX_set_ecdh_auto’ was not declared in this scope
16:06:26      if (SSL_CTX_set_ecdh_auto(ctx, 1) != 1)
16:06:26 
```
```
16:06:26  /usr/include/openssl/bio.h:668:6: note:   initializing argument 1 of ‘BIO* BIO_new_mem_buf(void*, int)’
16:06:26   BIO *BIO_new_mem_buf(void *buf, int len);
16:06:26        ^~~~~~~~~~~~~~~
16:06:26  /home/jenkins/workspace/Build_JDK8_x86-64_linux_Personal/build/linux-x86_64-normal-server-release/vm/compiler/../compiler/rpc/J9Server.cpp:148:35: error: invalid conversion from ‘const void*’ to ‘void*’ [-fpermissive]
16:06:26      BIO *certMem = BIO_new_mem_buf(&sslCerts[0][0], sslCerts[0].size());
```

**Comment by harryyu1994** [comment]
On it

**Comment by a7ehuo** [comment]
Likely the build still searches the default `/usr/include/` path instead of `/usr/local/include` which is where openssl-1.1.1 is installed, or search where the openssl is fetched and installed. 
```16:08:36  In file included from /usr/include/openssl/err.h:124:0```

This issue is related to issue #5720

Looks like the header versions that are picked up are prior to 1.0.2 
```
SSL_CTX_set_ecdh_auto
- Does not exist in 1.0.0, 1.0.1
- Declared in 1.0.2

BIO_new_mem_buf
- In 1.0.0 and 1.0.1
    - crypto/bio/bio.h:BIO *BIO_new_mem_buf(void *buf, int len);
- 1.0.2
    - include/openssl/bio.h:BIO *BIO_new_mem_buf(const void *buf, int len);
```

**Comment by harryyu1994** [comment]
Since SSL_CTX_set_ecdh_auto is deprecated, we should remove it. The rest of the issue should be fixed by #5720
